app/Http/Controllers/ReportsController.php

```diff
@@ -1110,13 +1110,19 @@ public function postAssetAcceptanceReport($deleted = false)
         $rows[] = implode(',', $header);
 
         foreach ($assetsForReport as $item) {
-            $row    = [ ];
-            $row[]  = str_replace(',', '', e($item['assetItem']->model->category->name));
-            $row[]  = str_replace(',', '', e($item['assetItem']->model->name));
-            $row[]  = str_replace(',', '', e($item['assetItem']->name));
-            $row[]  = str_replace(',', '', e($item['assetItem']->asset_tag));
-            $row[]  = str_replace(',', '', e(($item['acceptance']->assignedTo) ? $item['acceptance']->assignedTo->present()->name() : trans('admin/reports/general.deleted_user')));
-            $rows[] = implode(',', $row);
+
+            if ($item['assetItem'] != null){
```

Comment:
Should we add an `array_key_exists()` here too, to make sure `assetItem` is in the array?

Reply:
@inietov ping?

Reply:
I'm sorry, I overlooked your comment :S
I don't think it's necessary because we create those keys, so they always exist, just on some cases they doesn't get a value.